.yarn/versions/fc8f7ce5.yml

```diff
@@ -0,0 +1,29 @@
+releases:
+  "@yarnpkg/builder": prerelease
+  "@yarnpkg/check": prerelease
+  "@yarnpkg/cli": prerelease
+  "@yarnpkg/core": prerelease
+  "@yarnpkg/plugin-compat": prerelease
+  "@yarnpkg/plugin-constraints": prerelease
+  "@yarnpkg/plugin-dlx": prerelease
+  "@yarnpkg/plugin-essentials": prerelease
+  "@yarnpkg/plugin-exec": prerelease
+  "@yarnpkg/plugin-file": prerelease
+  "@yarnpkg/plugin-git": prerelease
+  "@yarnpkg/plugin-github": prerelease
+  "@yarnpkg/plugin-http": prerelease
+  "@yarnpkg/plugin-init": prerelease
+  "@yarnpkg/plugin-interactive-tools": prerelease
+  "@yarnpkg/plugin-link": prerelease
+  "@yarnpkg/plugin-npm": prerelease
+  "@yarnpkg/plugin-npm-cli": prerelease
+  "@yarnpkg/plugin-pack": prerelease
+  "@yarnpkg/plugin-patch": prerelease
+  "@yarnpkg/plugin-pnp": prerelease
+  "@yarnpkg/plugin-stage": prerelease
+  "@yarnpkg/plugin-typescript": prerelease
+  "@yarnpkg/plugin-version": prerelease
+  "@yarnpkg/plugin-workspace-tools": prerelease
+  "@yarnpkg/pnp": prerelease
+  "@yarnpkg/pnpify": prerelease
+  vscode-zipfs: prerelease
```

Comment:
Fwiw when updating the core in a backward-compatible way (such as adding an export), you don't need to mark all the plugins for releases. For example here I think we would just need to re-release `@yarnpkg/plugin-typescript` and `@yarnpkg/core`.

No big deal, but it helps me make the release faster (it takes ~an hour to make a release of everything, due to the various security checks)!